examples/kubernetes/templates/v1/cilium-crio-ds.yaml.sed

```diff
@@ -187,6 +187,35 @@ spec:
               add:
                 - "NET_ADMIN"
             privileged: true
+        - image: docker.io/cilium/cilium:latest
```

Comment:
The main cilium image on these templates uses` image: docker.io/cilium/cilium:__CILIUM_VERSION__` instead, which presumably picks up the version during template generation. I think we should do the same for these new containers.